packages/transport-webrtc/src/webrtc/rtc-data-channel.ts

```diff
@@ -0,0 +1,142 @@
+import { fromString as uint8ArrayFromString } from 'uint8arrays/from-string'
+import type node from 'node-datachannel'
+
+export class DataChannel extends EventTarget implements RTCDataChannel {
+  binaryType: BinaryType
+
+  readonly maxPacketLifeTime: number | null
+  readonly maxRetransmits: number | null
+  readonly negotiated: boolean
+  readonly ordered: boolean
+
+  onbufferedamountlow: ((this: RTCDataChannel, ev: Event) => any) | null
+  onclose: ((this: RTCDataChannel, ev: Event) => any) | null
+  onclosing: ((this: RTCDataChannel, ev: Event) => any) | null
+  onerror: ((this: RTCDataChannel, ev: Event) => any) | null
+  onmessage: ((this: RTCDataChannel, ev: MessageEvent) => any) | null
+  onopen: ((this: RTCDataChannel, ev: Event) => any) | null
+
+  #dataChannel: node.DataChannel
+  #bufferedAmountLowThreshold: number
+  #readyState: RTCDataChannelState
+
+  constructor (dataChannel: node.DataChannel) {
+    super()
+
+    this.#dataChannel = dataChannel
+    this.#readyState = 'connecting'
+
+    // TODO: what is the default?
+    this.#bufferedAmountLowThreshold = 0
```

Comment:
It can be removed I think. It's not in the spec and we don't set it anywhere.

Reply:
Because the spec lets you read the `bufferedAmountLowThreshold` property on a `RTCDataChannel` but the [DataChannel](https://github.com/murat-dogan/node-datachannel/blob/polyfill/lib/index.d.ts#L189-L205) object from `node-datachannel` only lets you set the value, not get it, so we cache it on the js side.

Reply:
Also, turns out the spec says the default is 0..